packages/typespec-go/test/cadlranch/authentication/apikeygroup/zz_responses.go

```diff
@@ -9,7 +9,7 @@ package apikeygroup
 
 // ApiKeyClientInvalidResponse contains the response from method ApiKeyClient.Invalid.
 type ApiKeyClientInvalidResponse struct {
-	InvalidAuth
```

Comment:
This is due to the updated TCGC which removed error cases from the non-error responses.